src/kreg/model.py

```diff
@@ -83,7 +87,9 @@ def fit(
             cg_maxiter=cg_maxiter,
             cg_maxiter_increment=cg_maxiter_increment,
             precon_build_freq=10,
+            disable_tqdm=disable_tqdm,
         )
 
         self.likelihood.detach()
+        self.fitted_result = result[0]
```

Comment:
Here It think it is probably good to save the entire result, just in case we need something later-on.